include/oneapi/dpl/pstl/hetero/dpcpp/parallel_backend_sycl_radix_sort.h

```diff
@@ -421,9 +434,14 @@ __radix_sort_scan_submit(_ExecutionPolicy&& __exec, ::std::size_t __scan_wg_size
             sycl::nd_range<1>(__radix_states * __scan_wg_size, __scan_wg_size), [=](sycl::nd_item<1> __self_item) {
                 // find borders of a region with a specific bucket id
                 sycl::global_ptr<_CountT> __begin = __count_rng.begin() + __scan_size * __self_item.get_group(0);
-                // TODO: consider another approach with use of local memory
-                sycl::ONEAPI::exclusive_scan(__self_item.get_group(), __begin, __begin + __scan_size, __begin,
-                                             _CountT(0), sycl::ONEAPI::plus<_CountT>{});
+            // TODO: consider another approach with use of local memory
+#if _ONEDPL_SYCL2020_COLLECTIVES_PRESENT
```

Comment:
I suggest using our own simple function-wrapper and place `_ONEDPL_SYCL2020_COLLECTIVES_PRESENT` logic within.
F.e. `__par_backend_hetero::exclusive_scan`